collectd/ping.sls

```diff
@@ -3,7 +3,12 @@
 include:
   - collectd
 
-liboping0:
+{% set ping_pkg = {
```

Comment:
Please move this logic to https://github.com/saltstack-formulas/collectd-formula/blob/master/collectd/map.jinja